dime-coding-standards/checklists/tex/reviewer_feedback_checklist.tex

```diff
@@ -94,18 +93,17 @@
 	
 		\vspace{.6em}
 
-        \section*{Reproducibility (applicable only if data was shared with you)}
+        \section*{Computational Reproducibility (applicable only if de-identified data was shared with you)}
 
-        \textit{\textbf{Next step:}} If you received data as part of the peer review package, attempt to run the code files. Ideally you will be able to do so by changing only the top-level directory in the master script. Take note of any reproducibility issues you observe as you will need to record them in \href{https://survey.wb.surveycto.com/collect/code_review_summary?caseid=}{this review summary form.}
+        \textit{\textbf{Next step:}} If you received data as part of the peer review package, attempt to run the code files. Ideally, you will be able to do so by changing only the top-level directory in the main script. Take note of any issues in reproducing code outputs (constructed indicators, cleaned data, graphs, and tables) as you will need to record them in \href{https://survey.wb.surveycto.com/collect/code_review_summary?caseid=}{this review summary form.}
```

Comment:
```suggestion
        \textit{\textbf{Next step:}} If you received data as part of the peer review package, attempt to run the code files. Ideally, you will be able to do so by changing only the top-level directory in the main script. 
```

Reply:
not necessary to link to form again, potentially confusing because makes it seem like it was a different form than linked at the top of the page